### ISSUE TITLE
Webshop shows price without tax on first load - in most browsers.

### ISSUE THREAD

**Issue by Lao-Tuo** [PrestaShop/PrestaShop#20227]
### Describe the bug

My shop is set to go "with tax".
When I open my shop first time on a new device (or private browsing window) the highlights module shows the Items but without tax. When I do a refresh, the price is correctly shown with tax.
The Item-preview in the highlights module also shows the Item correctly with tax.
I did a test on various systems and browsers:
 Linux:
    Firefox - no tax
    Chromium - no tax

iPhone:
    Safari - no tax

Windows (10):
    Firefox - no tax
    Chrome - no tax
    Edge (old) - with tax
    Edge (new) - with tax
 
#### Expected behavior

I would expect all browsers to show the highlights' price with tax

#### Steps to Reproduce

0. Open a private browsing window
1. Go to https://laotuo.de/de
2. See error
3. Refresh
4. Normal behavior

#### Additional information

* PrestaShop version: 1.7.6.7
* PHP version: 7.3

**Comment by Lao-Tuo** [comment]
So actually, that happens on directly going to the product as well. so it's not only the highlights.

Can't upload pictures for some reason.

**Comment by khouloudbelguith** [comment]
Hi @Lao-Tuo,

PrestaShop is not compatible with PHP 7.3, in fact, PrestaShop 1.7.0 to 1.7.3 is compatible with PHP 7.1 and PrestaShop 1.7.4 & 1.7.5 & PS1.7.6 are compatible with PHP 7.2.
For more details, you can follow this link: http://build.prestashop.com/news/prestashop-1-7-is-moving-to-symfony-3-4-and-php-5-6/
Next Major release PS1.7.7 will be compatible with PHP7.3.
You need to downgrade your PHP version.

Thanks to check and feedback.

**Comment by Lao-Tuo** [comment]
Hi,
Yeah.. I looked wrong in the chart.. :) https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/#php-compatibility-chart
I actually checked that, and had that 7 7 in my head.. missed that 6 in there.. and changed the php ver from 7.2 to 7.3.. my bad. 
anyways, now it's php 7.2 again.. same error..

also, it seems that the tax rule is simply not applied on the first load. the page source lists the property="product:pretax_price:amount" correct (for example 10.59), but has the property="product:price:amount" as net as well, instead of 12.60.
